modules/parquet/src/parquetjs/codecs/index.ts

```diff
@@ -20,5 +20,10 @@ export const PARQUET_CODECS: Record<ParquetCodec, ParquetCodecKit> = {
     // @ts-ignore
     encodeValues: PLAIN_DICTIONARY.encodeValues,
     decodeValues: PLAIN_DICTIONARY.decodeValues
+  },
+  RLE_DICTIONARY: {
+    // @ts-ignore
+    encodeValues: PLAIN_DICTIONARY.encodeValues,
```

Comment:
Some kind of comments here explaining why this would work? 

https://github.com/apache/parquet-format/blob/master/Encodings.md#dictionary-encoding-plain_dictionary--2-and-rle_dictionary--8

Reply:
Yes I added some comments.
` // Using the PLAIN_DICTIONARY enum value is deprecated in the Parquet 2.0 specification.`
` // Prefer using RLE_DICTIONARY in a data page and PLAIN in a dictionary page for Parquet 2.0+ files.`